changelogs/fragments/525.fixed

```diff
@@ -0,0 +1 @@
+Fixed a missing \` in the docs and added a little detail about the fact that the `.code-workspace` file is now an `.example`.
```

Comment:
The proper location for this file is the `changes/` directory.

Sorry for the confusion. The `changelogs/fragments` directory is a legacy location that the `anstibull-changelog` library uses, which we don't use in lieu of `towncrier`.

Reply:
Also, the filename should more aptly be named `525.documentation`.